content/en/blog/new-features-fission-v1.16.0.md

```diff
@@ -0,0 +1,121 @@
+
+To overcome this issue, the latest version introduced **Fission Services Check**.
+With a couple of basic commands, you can see the health of the environment and whether all the services are running as expected.
+You can now check the status of the environment using fission `check` command which will list down the status of Fission services.
+
+```bash
+$ fission check
+
+fission-services
+--------------------
+√ controller is running fine
+√ executor is running fine
+√ router is running fine
+× storagesvc deployment is not running
+
+fission-version
+--------------------
+√ fission is up-to-date
+```
+
+This is extremely helpful and handy while troubleshooting.
+Check our detailed guide on this in our [Troubleshooting Fission](docs/trouble-shooting/setup/fission/) document.
+
+### Commit Labels for Fission Spec
+
+One of the easiest ways to deploy your application Fission is using [Fission Spec](/docs/usage/spec/).
+It’s a collection of commands that instruct the Fission CLI to set up the environment, functions, packages etc. that is required to run an application.
+
+However, in practical scenarios, the application is never deployed in a single go.
+It’s usually iterated multiple times maybe due to feature additions or bug fixes.
+While using fission `spec apply` command, it wasn’t possible to **keep a track of which version of the code is actually deployed**.
+
+To overcome this, we’ve implemented a feature to append `--commitlabel` to `fission spec apply`.
+Whenever a fission spec apply command is executed, a label called `commit` is appended to all spec resources present in specdir.
+The value will be the `commitId` from the respective repository `HEAD`.
+
+The value of the `commit` label for different status of the file is as follows:
+
+| Git Gile Status                       | Label Value           |
+|---------------------------------------|-----------------------|
+| New untracked file                    | `untracked`           |
+| New staged file                       | `staged`              |
+| Tracked file with changes in worktree | `<commitID>-unstaged` |
+| Tracked file with changes staged      | `<commitID>-staged`   |
+| Tracked file with clean commit        | `<commitID>`          |
+
+### Fission Spec CLI Support
+
+A couple of minor feature additions were done to the Fission Spec CLI commands.
+
+One of the concerns was that when any Fission CRD is deleted and if the resources are not found, the command exits with non zero error code.
+Considering the **need for graceful exit**, we’ve added a new `--ignorenotfound` flag that returns a zero exit status even if the resource is not found.
+
+Another one is regarding Fission resources via specs.
+When a resource is already deployed with fission spec and we are trying to override it with some other deployed, then the command fails due to conflict.
+To allow this behavior, the `--force` flag was added that would allow you to override existing resources.
```

Comment:
And here To allow this behavior, the `--force` flag was added that would allow you to include the existing resources in the new deployment.